packages/dashboard/src/components/Wizard/Wizard.styles.ts

```diff
@@ -1,7 +1,73 @@
 import { IWizardStyles, IWizardStyleProps } from './Wizard.types';
+import { keyframes } from 'office-ui-fabric-react';
 
 export const subwayNavWidth = 303;
 export const subwayNavPadding = 48;
+export const wizardAnimationDurationMilliSec = 1000;

```

Comment:
We'll defnitely want to reduce this. I don't have explicit timings from Karina but I think we'll probably be more in the 250-300ms range.

Reply:
Lets do this when we sync up with Karina.

Reply:
I played around with some timing, 300 felt too fast.  I updated to 500.  But I think we can still adjust after review of the transitions.

Reply:
300 likely felt too fast if you didn't have the right easing function. Karina will be able to help there.